package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "space-invaders",
+  "version": "1.0.0",
+  "description": "Space flight is no joke, and neither is this lab.  You're going to need to create two JS constructors, `Spaceship` and `CrewMember`. These constructors will pump out instances that are by default unable to interact with one another.  A space ship will be inactive if it is created without a crew.  And a crew member by default will be hanging out in the Cantina looking for a ride if they were not put inside a ship when it is initialized.",
+  "main": "crewMember.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha -R mocha-multi --reporter-options nyan=-,json=.results.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/learn-co-curriculum/space-invaders.git"
+  },
+  "keywords": [
+    "js",
+    "classes",
+    "oop"
+  ],
+  "author": "scottcreynolds",
+  "license": "ISC",
```

Comment:
I might have missed this in other repos (I can update as needed, no need to go back), but the value of "license" should be "SEE LICENSE IN LICENSE.md".

Also, could we add an "engines" field?

```
"engines": {
    "node": "6.x",
    "npm": "3.x"
  },
```

Students'll need Node.js >= 6 :)